AFQ/segmentation.py

```diff
@@ -387,9 +387,17 @@ def _check_sl_with_inclusion(self, sl, include_rois, tol):
         dist = []
         for roi in include_rois:
             # Use squared Euclidean distance, because it's faster:
-            dist.append(cdist(sl, roi, 'sqeuclidean'))
-            if np.min(dist[-1]) > tol:
-                # Too far from one of them:
+            try:
```

Comment:
Maybe move this into a different PR? I think we should catch the case where the ROI is empty before we are here.